src/cmd/services/m3coordinator/ingest/carbon/ingest.go

```diff
@@ -67,7 +68,6 @@ var (
 
 // Options configures the ingester.
 type Options struct {
-	Debug             bool
```

Comment:
Btw may be prudent to put a deprecation warning [here](https://sourcegraph.com/github.com/m3db/m3@r/bootstrapping-changes/-/blob/src/cmd/services/m3query/config/config.go#L295:58) (unfortunately don't think we can delete the field without yaml parser complaining...)